### DISCUSSION TITLE
Is it possible for me to slide markdows without using vscode plugin or turning it into pdfs?

### DISCUSSION THREAD

**Discussion by calidion** [marp-team/marp#341] [Q&A] [answered]
Which can help me correct my slides' mistakes during slide presentaion.

**Comment by yhatt** [comment, marked as answer]
Did you try [Marp CLI's watch mode `--watch`](https://github.com/marp-team/marp-cli#watch-mode---watch---w-)?

**Reply by calidion** [reply]
Thanks, It would be great if can be integrated into vscode plugin.
